.storybook/main.js

```diff
@@ -6,4 +8,20 @@ module.exports = {
     '@storybook/addon-links',
     // '@whitespace/storybook-addon-html/register',
   ],
+
+  webpackFinal: async (config, { configType }) => {
+    // `configType` has a value of 'DEVELOPMENT' or 'PRODUCTION'
+    // You can change the configuration based on that.
+    // 'PRODUCTION' is used when building the static version of storybook.
+
+    // Make whatever fine-grained changes you need
+    config.module.rules.push({
+      test: /\.scss$/,
+      use: ['style-loader', 'css-loader', 'sass-loader'],
+      include: path.resolve(__dirname, '../'),
+    });
+
+    // Return the altered config
+    return config;
+  },
```

Comment:
I copied this from Storybook's docs:

https://storybook.js.org/docs/react/configure/webpack#extending-storybooks-webpack-config